src/patternfly/demos/PageLayout/page-template-header-brand-elements.hbs

```diff
@@ -0,0 +1,24 @@
+{{#if page-template-header-brand--toggle}}
+  {{#> button btnClass="pf-m-plain"
```

Comment:
we should probably update btnClass to be button--modifier. I'll create an issue for that though.